iron-icon.html

```diff
@@ -89,6 +100,7 @@
 
       width: var(--iron-icon-width, 24px);
       height: var(--iron-icon-height, 24px);
+      color: var(--iron-icon-color, inherit);
```

Comment:
@notwaldorf, it's not the same.
It works with `fill: currentcolor`. 
Right now we can colorize icons only via some color inheritance or by applying some global class like `.green { color: green }`. But with this change we will be able to apply **isolated**(encapsulated) **custom-styles** just like we do it with size (on the L28)